### ISSUE TITLE
guile-curl.pc is missing

### ISSUE THREAD

**Issue by NalaGinrut** [spk121/guile-curl#11]
It seems `guile-curl.pc` is missing. This causes issue in the downstream project. 

Here's a reference:
https://gitlab.com/hardenedlinux/artanis/-/issues/117

**Comment by NalaGinrut** [comment]
I think a typical template could be this:

```pc
prefix=@prefix@
includedir=${prefix}/include
libdir=${prefix}/lib

Name: @PACKAGE_NAME@
Description: PUT YOUR DESCRIPTION HERE.
Version: @PACKAGE_VERSION@
Cflags: -I${includedir}
Libs: -L${libdir} -l@PACKAGE_NAME@
Requires: guile-3.0
Requires.private:
```

You may need to name it as `guile-curl.pc.in`, and put it in `configure.ac` for generating actual pc file.

**Comment by jas4711** [comment]
FWIW, I've not seen any other guile package provide a *.pc file -- and C libraries are not expected to link to the libguile-curl library.  As suggested in the referenced bug report, a better solution is to use `guile.m4` to detect guile libraries from configure.  Thus, I suggest to close this as 'wontfix'.

**Comment by spk121** [comment]
I should mark this as won't fix, but, I left it open to remind me to write a better note on how to handle guile system extension paths in the docs, which I never did.
